# HG changeset patch (based on)
# User doko@ubuntu.com
# Date 1341068713 -7200
# Node ID d158b0a78390c104540c69d9d73f9c31fc5577dc
# Parent  575efae8ec5777b6130c29f7917d4e05303cfe12
- Issue #3754: Some unrelated configure.ac cleanups.

--- configure.orig	2015-01-21 11:43:28 UTC
+++ configure
@@ -9439,13 +9439,13 @@ fi
 if test -z "$with_pymalloc"
 then
     with_pymalloc="yes"
-    ABIFLAGS="${ABIFLAGS}m"
 fi
 if test "$with_pymalloc" != "no"
 then
 
 $as_echo "#define WITH_PYMALLOC 1" >>confdefs.h
 
+    ABIFLAGS="${ABIFLAGS}m"
 fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $with_pymalloc" >&5
 $as_echo "$with_pymalloc" >&6; }
